src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+$blueColor: #00acdc;
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $cardWidth;
```

Comment:
It seems like the variable `$cardWidth` is used here, but it is not defined in this file. Make sure that `$cardWidth` is defined somewhere in your styles or imported correctly.